src/components/ToDoWrapper.vue

```diff
@@ -0,0 +1,45 @@
+<template>
```

Comment:
Please add prettier and use prettier on this project with the following configuration: 
`
{
  "singleQuote": true,
  "useTabs": false,
  "tabWidth": 2,
  "vueIndentScriptAndStyle": true
}

`